app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -43,57 +43,155 @@ class BookmarksTest {
     @After
     fun tearDown() {
         mockWebServer.shutdown()
+        val bookmarksStorage = activityTestRule.activity?.bookmarkStorage
+        runBlocking {
+            val bookmarks = bookmarksStorage?.getTree(BookmarkRoot.Mobile.id)?.children
+            bookmarks?.forEach { bookmarksStorage.deleteNode(it.guid) }
+        }
+    }
+
+    @Test
+    fun noBookmarkItemsTest() {
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openLibrary {
+        }.openBookmarks {
+            verifyEmptyBookmarksList()
+        }
+    }
+
+    @Test
+    fun verifyBookmarkButtonTest() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+        }.openQuickActionBar {
```

Comment:
this is good for now, but I hear the quick action bar will be gone later this year.  We'll need to update the test when it happens.

Reply:
ok, I'll make sure to keep this up to date